include/h2o/socket.h

```diff
@@ -278,9 +279,11 @@ int32_t h2o_socket_getport(struct sockaddr *sa);
  * performs SSL handshake on a socket
  * @param sock the socket
  * @param ssl_ctx SSL context
+ * @param session_cache SSL session cache (maybe NULL)
```

Comment:
I would appreciate it if the type of the entries within the cache (i.e. `SSL_SESSION*`) would be clarified here.